app/views/hyrax/base/_attribute_rows.html.erb

```diff
@@ -8,7 +8,7 @@
 <%= presenter.attribute_to_html(:resource_type, render_as: :faceted, html_dl: true) %>
 <%= presenter.attribute_to_html(:medium, html_dl: true) %>
 <%= presenter.attribute_to_html(:source, render_as: :external_link, html_dl: true) %>
-<%= presenter.attribute_to_html(:technical_specs, html_dl: true) %>
-<%= presenter.attribute_to_html(:program_notes, html_dl: true) %>
+<%= presenter.attribute_to_html(:technical_specs, render_as: :markdown_generated) %>
```

Comment:
Should the "html_dl: true" be kept still?

Reply:
That makes sense. If all the other ones use it, then we should keep it in for consistency. I'll add the `html_dl: true back` in.